Automattic/woocommerce-payments
# Migrate Chip component to TypeScript to improve code quality

Previously part of #6723 – split into separate PR to simplify review.

#### Changes proposed in this Pull Request









This PR migrates the `Chip` component to TypeScript to improve code quality.

No visual or functional changes are expected.

#### Testing instructions





* Ensure GH checks pass with no changes to snapshot tests.
* View a `Chip` to ensure it renders correctly – for example the Account Status chip on **Payments → Overview**
	<img width="307" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/8d0efbdc-a290-4804-b49e-6861182288b7">


-------------------


**Post merge**
